src/components/settings/SafeAppsSigningMethod/index.tsx

```diff
@@ -0,0 +1,52 @@
+import ExternalLink from '@/components/common/ExternalLink'
+import { SETTINGS_EVENTS, trackEvent } from '@/services/analytics'
+import { useAppDispatch, useAppSelector } from '@/store'
+import { selectUseOnChainSigning, setUseOnChainSigning } from '@/store/settingsSlice'
+import { FormControlLabel, Checkbox, Paper, Typography, FormGroup, Grid } from '@mui/material'
+
+export const SafeAppsSigningMethod = () => {
+  const useOnChainSigning = useAppSelector(selectUseOnChainSigning)
+
+  const dispatch = useAppDispatch()
+
+  const onChange = () => {
+    trackEvent(SETTINGS_EVENTS.SAFE_APPS.CHANGE_SIGNING_METHOD)
+    dispatch(setUseOnChainSigning(!useOnChainSigning))
+  }
+
+  return (
+    <Paper sx={{ padding: 4 }}>
+      <Grid container spacing={3}>
+        <Grid item lg={4} xs={12}>
+          <Typography variant="h4" fontWeight="bold" mb={1}>
+            Signing method
+          </Typography>
+        </Grid>
+
+        <Grid item xs>
+          <Typography mb={2}>
+            This setting determines how your Safe Account will sign message requests from Safe Apps. By default it is
+            preferred to use gasless off-chain signatures. In case some Dapps only support on-chain signatures the
+            default behavior can be changed with this setting. Learn more about message signing{' '}
```

Comment:
I'm not sure about this description as it is a wallet-wide change for _all_ Safe Apps. I would simplify a bit (and explain that it is {Wallet}-based)

```suggestion
            This setting determines how the Safe{Wallet} will sign message requests from Safe Apps. Gasless, off-chain signing is used by default. Learn more about message signing{' '}
```